src/AppBundle/Controller/ProjectController.php

```diff
@@ -21,34 +22,52 @@ class ProjectController extends Controller
     const SERVER_ERROR                    = 'Server Error';
     const QUERY_CITY                      = 'city';
     const QUERY_PROJECT_ID                = 'project_id';
-    
+
     /**
-     * @Route("/projects", name="projects_list")
+     * @Route("/categories", name="categories_list")
```

Comment:
In my opinion better to rename it to "votings" and separate two metods bellow to its own controller (VotingController)